iina/PrefOSCToolbarSettingsSheetController.swift

```diff
@@ -91,11 +90,11 @@ class PrefOSCToolbarCurrentItem: NSImageView, NSPasteboardWriting {
   override func mouseDown(with event: NSEvent) {
     let dragItem = NSDraggingItem(pasteboardWriter: self)
     dragItem.draggingFrame = NSRect(origin: convert(event.locationInWindow, from: nil),
-                                    size: NSSize(width: 24, height: 24))
+                                    size: NSSize(width: Preference.ToolBarButton.frameWidth, height: Preference.ToolBarButton.frameHeight))
     dragItem.imageComponentsProvider = {
       let imageComponent = NSDraggingImageComponent(key: .icon)
       imageComponent.contents = self.buttonType.image()
-      imageComponent.frame = NSRect(origin: .zero, size: NSSize(width: 14, height: 14))
+      imageComponent.frame = NSRect(origin: .zero, size: NSSize(width: Preference.ToolBarButton.imageWidth, height: Preference.ToolBarButton.imageHeight))
```

Comment:
I'm not understanding why the need to reference Preference.ToolBarButton.imageWidth and imageHeight. At this point imageComponent.contents contains the NSImage. Instead of creating a NSSize, can't this just be imageComponent.contents.size, referencing the NSImage size property?